src/constants/currency.js

```diff
@@ -0,0 +1,45 @@
+import keyBy from 'lodash/keyBy';
+
+export const SYNTHS = [
```

Comment:
I don't think we should do that. Right now we're getting the synth list "dynamically" from synthetix-js. If we hardcode them then we need to maintain this list all the time when we add/remove synths from the system.